modules/va_facilities/app/controllers/va_facilities/v0/facilities_controller.rb

```diff
@@ -34,7 +34,8 @@ def all
       end
 
       def index
-        resource = BaseFacility.query(params).paginate(page: params[:page], per_page: params[:per_page])
+        resource = BaseFacility.query(params).paginate(page: params[:page],
+                                                       per_page: params[:per_page] || BaseFacility.per_page)
```

Comment:
Do we want to go ahead and start using `max_per_page` when it is smaller than `params[:per_page]`? We could also just create another ticket if you are ready to move on.

Reply:
Will go with a followup ticket so we can move on to the next ticket with this!